### PR TITLE
ClientCertificates description was wrong

### PR DESCRIPTION
**Pull request by rwatjen**
The ClientCertificates property description had text from the MaxResponseHeadersLength property.

## Summary

Updated the summary and value of HttpClientHandler.ClientCertificates

### CODE REVIEW

File: xml/System.Net.Http/HttpClientHandler.xml
Comment (JRAlexander) [inline xml/System.Net.Http/HttpClientHandler.xml]: See above about working with "ing".  We also usually add the type reference as a link.  How about something like "The \<see cref="T:System.Security.Cryptography.X509Certificates.X509CertificateCollection" /\>  that contains the security certificates associated with this request."?  Again, great catch!

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Great catch, @rwatjen! A few tweaks and we will be good to go. Thanks!

**Comment by rwatjen** [comment]
Hi @JRAlexander, I've updated with input from your comments. Since the HttpClientHandler can be reused in multiple requests, I haven't used your suggestions directly. I hope this will do :)

**Comment by JRAlexander** [comment]
@rwatjen, my apologies for the delay! I wanted to check on something with the devs and then was out of office late last week. Your changes are great! In the meantime, the API has been split into a new repo. So we are going to need to open a new PR there and copy your changes, and then close this PR. https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Net.Http/HttpClientHandler.xml
  
I've copied your changes here:
Summary - Gets the collection of security certificates that are associated requests to the server.
Value (removed the xref link, unnecessary after all) - The X509CertificateCollection that is presented to the server when performing certificate based client authentication.

After you copy those over, I'll merge straightaway after the build. Please let me know if you have any questions.

**Review by JRAlexander — APPROVED** [review]

**Comment by JRAlexander** [comment]
@rwatjen, Would you like me to do this? I can definitely if you'd rather. Please let me know asap. Thanks!

**Comment by rwatjen** [comment]
Hi @JRAlexander, I've created a new PR (https://github.com/dotnet/dotnet-api-docs/pull/37). Sorry for the delay, was busy at work.

**Pull request closed without merging**
